enterprise/models.py

```diff
@@ -1528,12 +1528,12 @@ def logo_path(instance, filename):
         filename (str): file to upload
 
     Returns:
-        path: path of image file e.g. enterprise/branding/<model.id>/<model_id>_logo.<ext>.lower()
+        path: path of image file e.g. enterprise/branding/logo_<uuid>.<ext>.lower()
 
     """
     extension = os.path.splitext(filename)[1].lower()
-    instance_id = str(instance.id)
-    fullname = os.path.join("enterprise/branding/", instance_id, instance_id + "_logo" + extension)
+    generated_uuid = str(uuid4())
```

Comment:
i really like that each upload gets a new uuid because it will help us deal with cache busting because each upload gets a unique asset url.